src/github/orbit-helpers.js

```diff
@@ -60,15 +46,21 @@ export const orbitAPI = {
    * @returns {is_a_member, contributions_collection, contributions_total}
    */
   async getMemberContributions(ORBIT_CREDENTIALS, username) {
+    const url = new URL(
+      `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/find`
+    );
+
+    const { params, headers } = configureRequest(ORBIT_CREDENTIALS, {
+      source: "github",
+      username: username,
+    });
+
+    url.search = params.toString();
+
     try {
-      const response = await fetch(
-        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/find?source=github&username=${username}&api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
-        {
-          headers: {
-            ...ORBIT_HEADERS,
-          },
-        }
-      );
+      const response = await fetch(url.toString(), {
```

Comment:
The `fetch()` global function [also accepts a `URL` as its first argument](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters), I think we can drop the `toString()`

Reply:
Aaaaah fixed, but context for you - this bad habit originally started because we had a test where we were checking the actual route that was requested (ie, StringContaining("/workspaces/members")`). I adopted `url.toString` as a default to maintain the original behaviour, but it turns out there was exactly 1 test where we did this. So... Totally unneeded haha